pkgs/by-name/gp/gpu-screen-recorder-gtk/package.nix

```
@@ -19,19 +19,19 @@
   wayland,
   wrapGAppsHook3,
   wrapperDir ? "/run/wrappers/bin",
+  gitUpdater,
 }:
 
-stdenv.mkDerivation (finalAttrs: {
+stdenv.mkDerivation rec {
   pname = "gpu-screen-recorder-gtk";
-  version = "5.0.0";
+  version = "5.1.2";
 
-  src = fetchurl {
-    url = "https://dec05eba.com/snapshot/gpu-screen-recorder-gtk.git.${finalAttrs.version}.tar.gz";
-    hash = "sha256-uXbiuA1XPWZVwQGLh47rKzCZSEUEPWqYALqMuCGA7do=";
+  src = fetchgit {
+    url = "https://repo.dec05eba.com/${pname}";
+    rev = version;
```

```suggestion
    tag = version;
```
https://github.com/NixOS/nixpkgs/pull/355973